config/routes.rb

```diff
@@ -51,6 +51,7 @@
   delete "/cart", to: "cart#empty"
   delete "/cart/:item_id", to: "cart#remove_item"
 
+  get "/profile/orders", to: "orders#index"
```

Comment:
Just to confirm, this route takes the `current_user` to an index page of all their orders, NOT an index page of every order on the site?

Reply:
That's right! I filtered the order with:

`Order.where(user_id: current_user.id)`